test/tests/parallel/set1/spam_virus/spam_junk_folder.bats

```diff
@@ -62,12 +62,16 @@ function teardown() { _default_teardown ; }
   )
   _init_with_defaults
   _common_container_setup 'CUSTOM_SETUP_ARGUMENTS'
+  _run_in_container sed -i -e 's/mailbox Junk/mailbox Spam/' /etc/dovecot/conf.d/15-mailboxes.conf
+  assert_success
+  _run_in_container dovecot reload
+  assert_success
```

Comment:
This should be handled via `user-patches.sh`:

- Store it in `test/config/junk-mailbox/user-patches.sh` similar to [this example](https://github.com/docker-mailserver/docker-mailserver/blob/d227d6dc7368274fa5c2b206e77b33274b4569bc/test/config/dovecot-lmtp/user-patches.sh). You will not need the `dovecot reload` this way, and it will better reflect testing the equivalent of where a user would introduce the change.
- Here is the associated test file](https://github.com/docker-mailserver/docker-mailserver/blob/d227d6dc7368274fa5c2b206e77b33274b4569bc/test/tests/parallel/set3/mta/lmtp_ip.bats#L26-L29) that uses that referenced `user-patches.sh`. Note how it's configured with the `mv` _before_ `_common_container_setup`, and _after_ `_init_with_defaults`.
- Use the blank lines like the reference example does for `mv`. Please include a brief comment for context to the reader so it's quicker to grok at a glance that [you're targeting this line](https://github.com/docker-mailserver/docker-mailserver/blob/d227d6dc7368274fa5c2b206e77b33274b4569bc/target/dovecot/15-mailboxes.conf#L54) / context. Suggested comment: "Adjust 'Junk' mailbox name to verify special-use flag delivers to modified mailbox folder name"

Reply:
Thank you for providing an example. I was not sure how to make changes to configuration files the best way, but found `_run_in_container` helper function and used it.

> Suggested comment: "Adjust 'Junk' mailbox name to verify special-use flag delivers to modified mailbox folder name"

Sounds a little weird to me. How about this?
"Adjust 'Junk' mailbox name to verify delivery to Junk mailbox based on special-use flag instead of mailbox's name"

Reply:
Yeah that revision is good too!

I guess you're just changing to `Spam` for the mailbox folder name to ensure it's not so similar to the special-use flag `\Junk`?